pyvista/report.py

```diff
@@ -98,6 +98,10 @@ def __repr__(self):
         content += '\n'
         return content
 
+    def __str__(self):
+        """Return string representation of GPUInfo."""
+        return f'GPUInfo(vendor={self.vendor}, renderer={self.renderer}, version={self.version})'
```

Comment:
In my mind, these sorts of representations (e.g., `Class(a=1, b=2)`) infer that the object can be instantiated but we never want anyone to populate these properties this themselves

Curious what the motivation here is?